src/routes/api/users.js

```diff
@@ -14,8 +17,8 @@ const router = require('express').Router();
  *            schema:
  *              $ref: '#/components/schemas/User'
  *      responses:
- *        "200":
+ *        "201":
  *          description: A user schema
```

Comment:
@william000000 would you mind documenting signup properly?

Reply:
@victorkarangwa4  Let me correct it